apps/sms/services/js/messaging/messaging_service.js

```diff
@@ -182,4 +188,8 @@ exports.MessagingService = BridgeServiceMixin.mixin(
   { methods: METHODS }
 );
 
+if ('importScripts' in self) {
+  MessagingService.init();  
+}
```

Comment:
If you want to know whether you're in a worker, we found that `if (!self.document)` is likely better